### ISSUE TITLE
Error appears when attempting to fetch browser history from specific browser using fetch [BUG]

### ISSUE THREAD

**Issue by SuvigyaJain1** [pesos/browser-history#28]
**Describe the bug**
Method for fetching specific browser history doesn't work
Error:
```bash
TypeError: fetch() missing 1 required positional argument: 'self'

```
**To Reproduce**
Followed the same steps as in README.
```python
from browser_history.browsers import Chrome
f = Chrome.fetch()
```

**Expected behavior**
No errors

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser Firefox
 - `browser-history` Version v0.2.0,
 - Python version:  3.7.2

**Comment by Samyak2** [comment]
Oops! That's a mistake in the documentation.

Browsers must be initialized before using them (instead of using the classes directly).
```python
from browser_history.browsers import Chrome

c = Chrome()

his = c.fetch().get()
```

**Comment by Samyak2** [comment]
Fixed in #29
